README.md

```diff
@@ -204,6 +205,24 @@ folder hierarchy.
 ⚠️ Copying objects (from S3 to S3) larger than 5GB is not supported yet. We have
 an [open ticket](https://github.com/peak/s5cmd/issues/29) to track the issue.
 
+#### Select JSON object content using SQL
+
+`s5cmd` supports the `SelectObjectContent` S3 operation, and will run your
+[SQL query](https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-glacier-select-sql-reference.html)
+against objects matching normal wildcard syntax and emit matching JSON records via stdout. Records
+from multiple objects will be interleaved, and order of the records is not guaranteed (though it's
+likely that the records from a single object will arrive in-order, even if interleaved with other
+records).
+
+    $ s5cmd select --compression GZIP \
+      --query "SELECT timestamp, hostname FROM S3Object WHERE ip_address LIKE '10.%' OR application='unprivileged'" \
+      s3://bucket-foo/object/2021/*
```

Comment:
I got an InvalidKeyPath error with this sample select query.
Query worked after updating as below:

```suggestion
    $ s5cmd select --compression GZIP \
      --query "SELECT s.timestamp, s.hostname FROM S3Object s WHERE s.ip_address LIKE '10.%' OR s.application='unprivileged'" \
      s3://bucket-foo/object/2021/*
```